src/test/java/org/apache/commons/crypto/stream/AbstractCipherStreamTest.java

```diff
@@ -313,7 +313,6 @@ protected CryptoCipher getCipher(String cipherClass) throws IOException {
 
     @Test
     public void testReadWrite() throws Exception {
-        Assert.assertEquals(null, Openssl.getLoadingFailureReason());
```

Comment:
Is it possible to make it a method of Openssl member method instead of removing the check here?